website/docs/r/synapse_workspace_keys.html.markdown

```diff
@@ -107,7 +109,7 @@ resource "azurerm_synapse_workspace_key" "example" {
   customer_managed_key_versionless_id = azurerm_key_vault_key.example.versionless_id
   synapse_workspace_id                = azurerm_synapse_workspace.example.id
   active                              = true
-  key_name                            = "enckey"
+  cusomter_managed_key_name           = "enckey"
```

Comment:
This field is deprecated so could we use `customer_managed_key_name` instead?

Reply:
Thanks! I didn't notice that type. I've fixed it, please take another look.